api/repositories/repositories_suite_test.go

```diff
@@ -268,6 +268,9 @@ func createAppCR(ctx context.Context, k8sClient client.Client, appName, appGUID,
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      appGUID,
 			Namespace: spaceGUID,
+			Annotations: map[string]string{
```

Comment:
Adding the annotation to `createAppCR` is not needed as your tests are not using this utility at all. In that case I would suggest reverting the method to keep the diff minimal.

Reply:
true. removed my changes